components/src/utils/AuthService.js

```diff
@@ -230,8 +227,10 @@ export function logoutAccountManagementUser() {
     }));
   }
   return logout().then(() => {
-    const keycloakLogoutRedirectUrl = `${Config.b2b.keycloak.logoutRedirectUrl}?redirect_uri=${encodeURIComponent(Config.b2b.keycloak.callbackUrl)}`;
-    window.location.href = keycloakLogoutRedirectUrl;
+    if (!Config.b2b.openId.enable) {
```

Comment:
i don't think we need this if statement, the above one returns something, so we shouldn't make it down here, unless this statement is already true 😄